### PR TITLE
Display sponsor cohort

### PR DESCRIPTION
**Pull request by personnamedmike**

### CODE REVIEW

**Comment by jeffreywdonahue** [comment]
needs review from two peeps @0x4d4n @alalonde @vidvidvid

File: pages/index.js [hunk 1]
```diff
@@ -292,9 +290,29 @@ export default function Home() {
...
+              minimumStake={minimumStake}
+              context={context}
+              raidBalance={raidBalance}
+              allowance={allowance}
+              isChecked={isChecked}
+              handleIsChecked={handleIsChecked}
+              cohortAddress={cohortAddress}
+              handleCohortAddress={handleCohortAddress}
+              isApproveTxPending={isApproveTxPending}
+              makeAnAllowance={makeAnAllowance}
```
Comment (alalonde) [inline pages/index.js:307]: These components have an awful lot of props, perhaps some of these could be added to a context?

**Review by alalonde — APPROVED** [review]
Looks good. I left an optional suggestion

**Comment by jeffreywdonahue** [comment]
thank you @alalonde one more and I can merge, @personnamedmike let me know if you want to revise.

**Comment by personnamedmike** [comment]
@alalonde @jeffreywdonahue I can refactor the code and add it to context. If it works correctly, should we just merge what we have now and I can make a new PR with the refactored code

**Comment by jeffreywdonahue** [comment]
Honestly yes at this point. Let's merge everything and get it tested.

I know it was a long weekend but we need to work on a better strategy.

**Pull request merged**